apps/web/Dockerfile

```diff
@@ -1,24 +1,55 @@
-FROM node:20-alpine AS build
+FROM node:20-alpine AS base
 
+
+FROM base AS builder
+RUN apk add --no-cache libc6-compat
+RUN apk update
```

Comment:
Steps that update, use `apk update && apk upgrade` together

Reply:
resolved